src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -104,3 +104,35 @@
     text-decoration: none;
   }
 }
+
+@mixin gradientStyle($stopColor, $startColor: $color-ink-80) {
+  background-image: linear-gradient($startColor, $stopColor);
+
+  .HeroRecommendation-link {
+    &:hover,
+    &:active {
+      background-color: $stopColor;
+    }
+  }
+}
+
+// Styles that correspond to gradients returned by the hero API
+.color-ink-80-color-blue-70 {
```

Comment:
These all need to be prefixed with the component name, like `.HeroRecommendation-color-ink-80-color-blue-70`.